package.json

```diff
@@ -401,6 +409,16 @@
         "title": "Hide Generated",
         "category": "ObjectScript"
       },
+      {
+        "command": "vscode-objectscript.explorer.showSystem",
+        "title": "Show System",
+        "category": "ObjectScript"
+      },
+      {
+        "command": "vscode-objectscript.explorer.hideSystem",
+        "title": "Hide System",
```

Comment:
Since %Z classes aren't system classes, I suggest this menu item name be "Hide % Classes"

reference: https://docs.intersystems.com/irislatest/csp/docbook/Doc.View.cls?KEY=GORIENT_appx_identifiers